### ISSUE TITLE
Ghosting blocks on servers with lag

### ISSUE THREAD

**Issue by sefirosweb** [PrismarineJS/mineflayer#2600]
- [x] The [FAQ](https://github.com/PrismarineJS/mineflayer/blob/master/docs/FAQ.md) doesn't contain a resolution to my issue 

## Versions
 - mineflayer: 4.2.0
 - server: spigot 1.18.1
 - node: 16

## Detailed description of a problem
When the server is a bit slow or does overflow sometimes the blocks are Ghosting

For exaple this happens in the client of legacy when are you playing
But i see this occurs more often to bots,
D you have same issue?

![image](https://user-images.githubusercontent.com/20754836/164565314-35395144-1bc7-4426-9d5c-085f38be669d.png)

Example:
https://www.youtube.com/watch?v=oFKcJfI138Y
Minute 1:25, the server is laggy and the bot skip the block

**Comment by alexanderbkl** [comment]
Any updates?

**Comment by alexanderbkl** [comment]
I think the only thing that can fix it is to update the rendering by right-clicking always in case there is a lag spike as such.
I remember this happened a lot on laggy servers and I used to mine with left click and right click at the same time.

**Comment by sefirosweb** [comment]
> I think the only thing that can fix it is to update the rendering by right-clicking always in case there is a lag spike as such. I remember this happened a lot on laggy servers and I used to mine with left click and right click at the same time.

Yep i upgrade the server with more cpu and ram and had fixed a bit, is that you say when server is slow they make a ghostint blocks

I think they don't have solution because the bot doesn't know the block is a fake block
